src/test/java/service/TestDataReader.java

```diff
@@ -0,0 +1,11 @@
+package service;
+
+import java.util.ResourceBundle;
+
+public class TestDataReader {
+    private static ResourceBundle resourceBundle = ResourceBundle.getBundle(System.getProperty("environment", "qa"));
```

Comment:
I suppose, everywhere data will be get just from qa env....
but we need also from 'dev' env....

Reply:
both properties are used